server/src/services/dataService.py

```diff
@@ -18,12 +19,15 @@ def standardFilters(self,
         '''
         Generates filters for dates, request types, and ncs.
         '''
-        return [
-            Request.createddate > startDate if startDate else False,
-            Request.createddate < endDate if endDate else False,
-            Request.requesttype.in_(requestTypes),
-            Request.nc.in_(ncList),
-        ]
+
+        requestTypes = (', ').join([f"'{rt}'" for rt in requestTypes])
```

Comment:
Deja vu 😂

Reply:
I know lol. I was all about the ORM for a minute (sheep emoji), but it's actually kind of annoying. Documentation isn't great, plus it's just an extra code layer that we really don't need for this app.